build.gradle

```diff
@@ -89,9 +102,12 @@ idea {
 }
 
 dependencies {
-    provided 'org.jruby:jruby-core:1.7.22'
-    provided files('lib/logstash-core-event-java.jar')
+    provided 'org.jruby:jruby-core:1.7.25'
+    provided files('java_libs/logstash-core-event-java.jar')
```

Comment:
we need to figure out a better way of dealing with this. For now, we should document what you need to do to have this file be provided